src/MultivariateStats.jl

```diff
@@ -282,7 +281,7 @@ For more information about the algorithm, see the paper by Li, Zhu and Ogihara,
     cov_b::CovarianceEstimator = MS.SimpleCovariance()
     out_dim::Int     = 0::(_ ≥ 0)
     regcoef::Real    = 0.0::(_ ≥ 0)
-    dist::SemiMetric = SqEuclidean()
+    priors::Union{Nothing, MLJBase.UnivariateFinite} = nothing
```

Comment:
Let's not use union types if we don't need to, here nothing is the same as providing a UnivariateFinite with uniform probabilities on all classes

Reply:
But there are times the user is lazy and wants these priors automatically estimated. Should i still remove this from feature code?